### PR TITLE
[WIP] Fixes type errors

### PR DESCRIPTION
**Pull request by madsnedergaard**

### CODE REVIEW

File: web/src/features/charts/bar-breakdown/utils.ts [hunk 1]
```diff
@@ -131,29 +115,30 @@ export const getDataBlockPositions = (
...
+  exchangeKeys: ZoneKey[],
+  electricityMixMode: MixMode
 ): ExchangeDataType[] =>
-  exchangeKeys.map((mode) => {
+  exchangeKeys.map((key) => {
     // Power in MW
-    const exchange = (data.exchange || {})[mode];
-    const exchangeCapacityRange = (data.exchangeCapacities || {})[mode];
+    const exchange = (data.exchange || {})[key];
+    const exchangeCapacityRange = (data.exchangeCapacities || {})[key];
```
Comment (VIKTORVAV99) [inline web/src/features/charts/bar-breakdown/utils.ts:131]: Shouldn't this use optional chaining instead?
```suggestion
    const exchange = data.exchange?.[key];
    const exchangeCapacityRange = data.exchangeCapacities?.[key];
```

**Review by VIKTORVAV99 — COMMENTED** [review]
One tiny suggestion/question 🙂

**Comment by madsnedergaard** [comment]
Closing this again - let's address it from "fresh" after the merge. Will cherry pick some of the commits from here after we have merged in :)

**Comment by madsnedergaard** [comment]
Closing this again - let's address it from "fresh" after the merge. Will cherry pick some of the commits from here after we have merged in :)

**Pull request closed without merging**